python/paddle/fluid/layers/nn.py

```diff
@@ -2282,40 +2282,67 @@ def sequence_conv(input,
 
 def sequence_softmax(input, use_cudnn=False, name=None):
     """
-    This function computes the softmax activation among all time-steps for each
-    sequence. The dimension of each time-step should be 1. Thus, the shape of
-    input Tensor can be either :math:`[N, 1]` or :math:`[N]`, where :math:`N`
-    is the sum of the length of all sequences.
+    **Note**:
+    
+    **The input type of the OP must be LoDTensor. If the input type to be processed is Tensor, use:** :ref:`api_fluid_layers_softmax` 
```

Comment:
If the input type to be processed is Tensor
=>
For Tensor

Reply:
done